ffi/tests/ctests/src/tests.c

```diff
@@ -0,0 +1,219 @@
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+
+#include <wirefilter.h>
+
+extern void rust_assert(bool check, const char *file, size_t line, const char *msg);
+
+#define assert(check, msg) rust_assert(check, __FILE__, __LINE__, msg)
+
+static wirefilter_externally_allocated_str_t wirefilter_string(const char *s) {
+	wirefilter_externally_allocated_str_t str = {(const unsigned char *)s, strlen(s)};
```

Comment:
You can return value inline using `return (wirefilter_externally_allocated_str_t) { ... }`.

Also `unsigned` cast shouldn't be necessary, you can change type definition to use regular `char` for simplicity (they're interchangeable ABI-wise).

Reply:
Ok I will make the changes.